src/node/interop/interop_client.js

```diff
@@ -135,7 +135,7 @@ function largeUnary(client, done) {
 function clientStreaming(client, done) {
   var call = client.streamingInputCall(function(err, resp) {
     assert.ifError(err);
-    assert.strictEqual(resp.aggregated_payload_size, 74922);
+    assert.strictEqual(resp.aggregatedPayloadSize, 74922);
```

Comment:
Protobuf.JS spits out camel case now, so I had to change this in a number of places. `bytes_field` never worked in place of `bytesField` when encoding. It made sense then to change all of the relevant fields to camel case.